tests/docker/install/_helper_run_install_tests.sh

```diff
@@ -30,8 +30,8 @@ echo "Installing test dependencies for Python $python_version on $pretty_name"
     if grep Alpine /etc/os-release; then
         # test deps
         apk add bash
-        # agent deps - APM dep on psutil, so Alpine need linux-headers
-        apk add python3 curl linux-headers
+        # agent deps - APM deps on psutil, so Alpine needs more deps
+        apk add python3 curl linux-headers gcc musl-dev
```

Comment:
We have to add more deps back 🙃 but only for Alpine, in order to install dep `psutil`. See also https://github.com/solarwinds/apm-python/pull/633